openmdao_book/features/experimental/plugins.md

```diff
@@ -0,0 +1,58 @@
+# Working with Plugins
+
+The OpenMDAO plugin infrastructure provides a way for you to discover and use code that can extend the functionality of OpenMDAO.
+
+## Discovery of Plugins
+
+Before you can use a plugin, you have to know that it exists,  There are a couple of places on the web where you might find OpenMDAO plugins.  The first is [github](https://github.com). Github repositories can have *topics* associated with them.  These are basically like keywords.  You can type a topic into the github search field at the top of the main page.  If you choose `All Github` as your search scope, you should then see a page with a table showing different searching sub-scopes.  Look for `Topics` in that table and select it.  The topic will be displayed on the right along with the number of repositories found that match that topic.  If you then click on that topic link you'll be taken to a page containing a list of links to repositories that have that topic.  The table below gives a list of openmdao-related topics and what type of plugin a repository labelled with that topic should contain:
+
+
+    | Topic Search String       | Repo Should Contain       |
+    | ------------------------- | ------------------------- |
+    | openmdao                  | anything openmdao related |
+    | openmdao_component        | Component                 |
+    | openmdao_group            | Group                     |
+    | openmdao_driver           | Driver                    |
+    | openmdao_lin_solver       | LinearSolver              |
+    | openmdao_nl_solver        | NonlinearSolver           |
+    | openmdao_surrogate_model  | SurrogateModel            |
+    | openmdao_case_recorder    | CaseRecorder              |
+    | openmdao_case_reader      | BaseCaseReader            |
+    | openmdao_command          | command line tool         |
+
+
+The [openmdao find_plugins](../../other_useful_docs/om_command.ipynb#openmdao-find_plugins) command was created to make this process a little easier.  To use the tool, just specify the type of plugin you're looking for, or you can look for anything associated with openmdao by just specifying 'openmdao'. The allowed plugin types are any of the topic search strings shown in the table above with the *openmdao_* prefix removed, so for example, you could look for drivers using the following command:
+
+```
+openmdao find_plugins driver
+```
+
+The command will display the name and URL of any repositories that it finds.  Depending upon the structure of a given repository, it may be possible to install the python package contained in that repository directly from github using only the URL.  For example:
+
+```
+    pip install git+https://github.com/naylor-b/om_devtools.git
```

Comment:
Small issue but this block and the block below it should be tabbed back to be even with the other blocks. The only one that gets tabbed in is autoX rst methods

Reply:
fixed